### PR TITLE
Adapt mgr-create-bootstrap-repo to create bootstrap repo for Ubuntu

### PR DESCRIPTION
**Pull request by mateiw**
## What does this PR change?
 Adapt `mgr-create-bootstrap-repo` to create bootstrap repo for Ubuntu.

## GUI diff

No difference.

## Documentation
- No documentation needed: **add explanation. This can't be used if there is a GUI diff**
- [doc-susemanager](https://github.com/SUSE/doc-susemanager) PR or issue was created (GitHub automatic link expected below)

- [ ] **DONE**

## Test coverage
- No tests: **add explanation**
- Unit tests were added
- Cucumber tests were added

- [ ] **DONE**

## Links

Fixes https://github.com/SUSE/spacewalk/issues/6151

- [ ] **DONE**

## Changelogs

If you don't need a changelog check, please mark this checkbox:

- [ ] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_lint_checkstyle"		 
- [ ] Re-run test "java_pgsql_tests"		 
- [ ] Re-run test "ruby_rubocop"
- [ ] Re-run test "schema_migration_test_oracle"
- [ ] Re-run test "schema_migration_test_pgsql"		 
- [ ] Re-run test "susemanager_unittests"

### CODE REVIEW

File: susemanager/src/mgr_bootstrap_data.py [hunk 1]
```diff
@@ -353,6 +353,34 @@
     "libunwind",
 ]
 
+PKGLISTUBUNTU = [
+    "python3-tornado",
```
Comment (mcalmer) [inline susemanager/src/mgr_bootstrap_data.py]: Python 3? I thought salt is compiled for python 2 in Ubuntu. Please check also if the list is the same for 16.04 and 18.04

File: susemanager/src/mgr_bootstrap_data.py [hunk 2]
```diff
@@ -353,6 +353,34 @@
...
+    "python-zmq",
+    "libpgm-5.2-0",
+    "python-zmq-dbg",
+    "libzmq3-dev",
+    "libpgm-dev",
+    "python3-systemd",
+    "python3-zmq",
+    "libzmq5",
+    "python-systemd",
+    "salt-master",
```
Comment (mcalmer) [inline susemanager/src/mgr_bootstrap_data.py]: We only ship salt-common and salt-minion. Master, api, ssh, syndic or cloud are not needed.

File: susemanager/src/mgr_bootstrap_data.py [hunk 3]
```diff
@@ -682,4 +710,14 @@
         'BASECHANNEL' : 'centos7-x86_64', 'PKGLIST' : RES7,
         'DEST' : '/srv/www/htdocs/pub/repositories/centos/7/bootstrap/'
     },
+    'ubuntu-16.04-amd64' : {
+        'BASECHANNEL' : 'ubuntu-16.04-amd64-main', 'PKGLIST' : PKGLISTUBUNTU,
```
Comment (mcalmer) [inline susemanager/src/mgr_bootstrap_data.py]: We have product ids for Ubuntu. We could use the standard configuration similar to SLE15 .
But we need to call it allways with `--with-custom-channels`. So I am a bit unsure what type is better.

File: susemanager/src/mgr_bootstrap_data.py [hunk 4]
```diff
@@ -352,6 +352,86 @@
     "libunwind",
 ]
 
+PKGLISTUBUNTU1604 = [
+    "libsodium18",
```
Comment (mbologna) [inline susemanager/src/mgr_bootstrap_data.py]: Please notice that some of the packages listed here have a different name in the specfile in OBS, e.g. `libsodium`
Would that be a problem?

File: susemanager/src/mgr_bootstrap_data.py [hunk 5]
```diff
@@ -681,4 +761,14 @@
         'BASECHANNEL' : 'centos7-x86_64', 'PKGLIST' : RES7,
         'DEST' : '/srv/www/htdocs/pub/repositories/centos/7/bootstrap/'
     },
+    'ubuntu-16.04-amd64' : {
+        'BASECHANNEL' : 'ubuntu-16.04-amd64-main', 'PKGLIST' : PKGLISTUBUNTU1604,
```
Comment (mcalmer) [inline susemanager/src/mgr_bootstrap_data.py]: Wrong base channel name. The name we use now is: `ubuntu-16.04-pool-amd64`

File: susemanager/src/mgr_bootstrap_data.py [hunk 6]
```diff
@@ -681,4 +761,14 @@
...
+        'DEST' : '/srv/www/htdocs/pub/repositories/ubuntu/16/4/bootstrap/',
+        'TYPE' : 'deb'
+    },
+    'ubuntu-18.04-amd64' : {
+        'BASECHANNEL' : 'ubuntu-18.04-amd64-main', 'PKGLIST' : PKGLISTUBUNTU1804,
```
Comment (mcalmer) [inline susemanager/src/mgr_bootstrap_data.py]: Wrong base channel name. The name we use now is: `ubuntu-18.04-pool-amd64`

File: susemanager/src/mgr_bootstrap_data.py (hunk 4 above)
Comment (mateiw) [reply susemanager/src/mgr_bootstrap_data.py]: It's not a problem. In the repo the binaries are called `libsodium18` and `libsodium23`. In the `control` file the name is set explicitly to `Package: libsodium23` for e.g.

File: susemanager/src/mgr_bootstrap_data.py (hunk 6 above)
Comment (mateiw) [reply susemanager/src/mgr_bootstrap_data.py]: Changed base channel to `ubuntu-18.04-pool-amd64`

File: susemanager/src/mgr_bootstrap_data.py (hunk 5 above)
Comment (mateiw) [reply susemanager/src/mgr_bootstrap_data.py]: Changed base channel to `ubuntu-16.04-pool-amd64`

File: susemanager/src/mgr_bootstrap_data.py (hunk 1 above)
Comment (mateiw) [reply susemanager/src/mgr_bootstrap_data.py]: Package lists are now separate for 16.04 and 18.04.

**Pull request merged**